core/src/OpenACC/Kokkos_OpenACC_ParallelFor_Team.hpp

```diff
@@ -45,7 +45,6 @@
 #ifndef KOKKOS_OPENACC_PARALLEL_FOR_TEAM_HPP
 #define KOKKOS_OPENACC_PARALLEL_FOR_TEAM_HPP
 
-#include <openacc.h>
```

Comment:
Dropping it because it is not actually needed for `#pragma acc` directives, only when calling runtime routines or referring to typedefs or constants which we don't do in that header file?

Reply:
`#include <openacc.h>` is dropped because it is already included in `OpenACC/Kokkos_OpenACC_Team.hpp`.